build.gradle

```diff
@@ -38,19 +38,41 @@ subprojects {
     }
 }
 
-project(":TweetWallFX-Core"){
+project (":TweetWallFX-CommandlineArguments") {
+    dependencies {
+        compile group: 'org.netbeans.api', name: 'org-openide-util-lookup', version: 'RELEASE80'
+        compile group: "com.beust", name: "jcommander", version: "1.48"
+    }
+}
+
+project(":TweetWallFX-Tweet-API") {
+    dependencies {
+        compile group: 'org.netbeans.api', name: 'org-openide-util-lookup', version: 'RELEASE80'
+    }
+}
+
+project(":TweetWallFX-Tweet-SPI-Twitter") {
```

Comment:
Is spi-twitter the correct naming? Probably impl-twitter4j would be more exact. SPI should be reserved for an spi not an implementation.